.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Gradle Plugin Publish
+on:
+  release:
+    types:
+      - published
```

Comment:
`published` event is quite tricky that is not triggered when a release is made by a bot.
https://github.com/line/centraldogma/pull/844
What do you think of using `on.push.tag` event instead?